oeps/oep-0013.rst

```diff
@@ -0,0 +1,576 @@
+====================================
+OEP-13: Open edX web API conventions
+====================================
+
+.. This is the template to use when you start a new OEP.
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-13 </oeps/oep-0013>`            |
++---------------+-------------------------------------------+
+| Title         | Open edX web API conventions              |
++---------------+-------------------------------------------+
+| Last Modified | 2016-12-29                                |
++---------------+-------------------------------------------+
+| Author        | Edward Fagin <eddie@edx.org>              |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+|  Created      | 2016-12-29                                |
++---------------+-------------------------------------------+
+| `Resolution`  |                                           |
++---------------+-------------------------------------------+
+| `Replaces`    |                                           |
++---------------+-------------------------------------------+
+| `Replaced-By` |                                           |
++---------------+-------------------------------------------+
+| `References`  |                                           |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+This document captures the conventions to be used for all Open edX web APIs.
+
+Motivation
+==========
+
+The Open edX platform contains many extension points for third-party developers that wish to integrate with the platform. We wish to approach a standard methodology for API design in order to give developers more consistency in their API client design, and to make our endpoint behaviors more predictable.
```

Comment:
Nit: we settled on wrapping lines at 80 characters in OEPs. 

From @cpennington: 
> I tend to prefer wrapped lines in the raw text, simply because it's easier to read in an editor (and in the email summaries that github sends). Github wraps them nicely in the diff, but in this case I lean towards optimizing towards the lowest common denominator.